variables.tf_TEMPLATE

```diff
@@ -11,6 +11,14 @@ variable "cred_file" {
   default = "/Users/yourname/.aws/credentials-gamingaws"
 }
 
+# Update this if you are using a single credential file
```

Comment:
I actually tried doing this, but had trouble, which is why I used a non-default credentials file.  But, it's a good idea to include this.